packages/fxa-settings/src/components/Settings/FlowSetup2faPrompt/index.tsx

```diff
@@ -0,0 +1,94 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import React from 'react';
+import { FtlMsg } from 'fxa-react/lib/utils';
+import { BackupRecoveryPhoneCodeImage } from '../../images';
+import LinkExternal from 'fxa-react/components/LinkExternal';
+import FlowContainer from '../FlowContainer';
+import { GleanClickEventType2FA } from '../../../lib/types';
+import Banner from '../../Banner';
+
+export type FlowSetup2faPromptProps = {
+  localizedPageTitle: string;
+  onContinue: () => void;
+  onCancel: () => void;
+  serviceName: string;
+  localizedErrorMessage?: string;
+};
+
+export const FlowSetup2faPrompt = ({
+  localizedPageTitle,
+  onContinue,
+  onCancel,
+  serviceName,
+  localizedErrorMessage,
+}: FlowSetup2faPromptProps) => {
+  return (
+    <FlowContainer onBackButtonClick={onCancel} title={localizedPageTitle}>
```

Comment:
UX question - I wonder if a back button makes sense on this screen or should be omitted. Makes sense on following screens, but not sure about this one